django/db/backends/oracle/base.py

```diff
@@ -200,10 +200,13 @@ def _connect_string(self):
                              settings_dict['PASSWORD'], dsn)
 
     def get_connection_params(self):
+        # Specify CHAR and NCHAR encoding.
```

Comment:
I think the "what" is obvious without a comment. Commenting about the "why" is what's more important, I think.